### DISCUSSION TITLE
StarSolo ATACseq genome Index

### DISCUSSION THREAD

**Discussion by akhst7** [alexdobin/STAR#1810] [Q&A]
Hi, 

Kallisto bus tools uses "the DNase I Hypersensitive Sites interval list" to generate Kb ref index for ATACseq.   Could anybody share tricks to generate StarSolo genome index for ATACseq ? 

Thanks.

**Comment by alexdobin** [comment]
Hi @akhst7 

with STAR, you can map ATAC-seq to the whole genome, no need to generate a special index.

**Comment by akhst7** [comment]
Thanks.   

What I really wanted to ask was the way to generate a fragment count matrix from genome mapped bam in addition to SnapATAC (#558), which now is SnapATAC2 in python. 

Hopefully, the future StarSolo can generate a fragment the count matrix or perhaps a fragment file in the bed format.

**Reply by alexdobin** [reply]
That's something I am thinking about for the future development.
